### ISSUE TITLE
Can this plugin change the class name in my html and js file?

### ISSUE THREAD

**Issue by lamualfa** [JPeer264/rcs-webpack-plugin#3]
Thank you very much. You have made a very helpful library. I really appreciate what you have made. But sorry, can you provide more examples code for using this library? can this library be integrated with the [html-webpack-plugin](https://github.com/jantimon/html-webpack-plugin) plugin? can this plugin automatically change the entire class name in the `html` and` js` files that I specified in the `entry` options on the [webpack](https://github.com/webpack/webpack)?

**Comment by JPeer264** [comment]
Hey thanks for your issue. All your files should automatically change. I think it should also work with the `html-webpack-plugin` as this plugin generates the html file and transfers it into the chunks (unless you ignore the files with, e.g., the `excludeChunks`  option mentioned [here](https://github.com/jantimon/html-webpack-plugin#options)). Just make sure the `rcs-webpack-plugin` is the last array entry in your webpack config.

Would it help if I also put an example including the usage with `html-webpack-plugin`. Any other example suggestions?

**Comment by lamualfa** [comment]
> Hey thanks for your issue. All your files should automatically change. I think it should also work with the `html-webpack-plugin` as this plugin generates the html file and transfers it into the chunks (unless you ignore the files with, e.g., the `excludeChunks` option mentioned [here](https://github.com/jantimon/html-webpack-plugin#options)). Just make sure the `rcs-webpack-plugin` is the last array entry in your webpack config.
> 
> Would it help if I also put an example including the usage with `html-webpack-plugin`. Any other example suggestions?

```js
require('dotenv').config();

const autoprefixer = require('autoprefixer');
const config = require('config');
const path = require('path');

const HtmlWebpackPlugin = require('html-webpack-plugin');
const MiniCssExtractPlugin = require('mini-css-extract-plugin');

const { CleanWebpackPlugin } = require('clean-webpack-plugin');
const { default: RcsWebpackPlugin } = require('rcs-webpack-plugin');

const srcDir = path.resolve(config.get('app.srcDir'));
const viewDir = path.resolve(config.get('app.viewDir'));
const publicDir = path.resolve(config.get('app.publicDir'));

module.exports = {
  entry: {
    'page/test': path.join(srcDir, 'view/page/test/test.js')
  },
  output: {
    path: path.join(publicDir, 'statics'),
    filename: 'js/[name].js',
    publicPath: 'statics'
  },
  module: {
    rules: [
      {
        test: /\.(sa|sc|c)ss$/,
        use: [
          {
            loader: MiniCssExtractPlugin.loader,
            options: {
              hmr: process.env.NODE_ENV === 'development'
            }
          },
          {
            loader: 'css-loader'
          },
          {
            loader: 'postcss-loader',
            options: {
              ident: 'postcss',
              plugins: [autoprefixer]
            }
          },
          'sass-loader'
        ]
      }
    ]
  },
  plugins: [
    new HtmlWebpackPlugin({
      filename: path.join(viewDir, 'page/test.hbs'),
      template: path.join(srcDir, 'view/page/test/test.hbs')
    }),
    new MiniCssExtractPlugin({
      filename: 'css/[name].css',
      chunkFilename: 'css/[id].css'
    }),
    new RcsWebpackPlugin({
      fillLibrariesOptions: true
    }),
    new CleanWebpackPlugin()
  ]
};

```

Rcs only change the class names in `css` file, it doesn't work in `html` and `js` file.

**Comment by JPeer264** [comment]
Did you also try to add the`handlebars-loader` as written in the [docs](https://github.com/jantimon/html-webpack-plugin#writing-your-own-templates)?

**Comment by lamualfa** [comment]
> Did you also try to add the`handlebars-loader` as written in the [docs](https://github.com/jantimon/html-webpack-plugin#writing-your-own-templates)?

Not just `handlebars`, but I've tried using plain `html`. but still can't. Can you include some examples of usage with normal `html`?

**Comment by JPeer264** [comment]
Alright, I will investigate into this :)

**Comment by JPeer264** [comment]
I was wrong initially, it seems that `html-webpack-plugin` has implemented its own loader and things work differently.

I published now v`1.1.0` with `html-webpack-plugin` compatibility. But I think I need to implement handlebars into `rcs-core`, this may take a while but I keep you updated :)

**Comment by JPeer264** [comment]
Sorry that it took so long. But in v`1.2.0` everything should work now. Also, I added some [examples](https://github.com/JPeer264/rcs-webpack-plugin/tree/master/examples)
